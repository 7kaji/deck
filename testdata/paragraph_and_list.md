# Title

**C**
D
- A
- B

<!-- {"layout":"title-and-body"} -->
