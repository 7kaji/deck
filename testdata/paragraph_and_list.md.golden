[
  {
    "layout": "title-and-body",
    "titles": [
      "Title"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "C ",
                "bold": true
              },
              {
                "value": "D"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "A"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "B"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  }
]